src/main/java/picard/illumina/IlluminaBasecallsToSam.java

```diff
@@ -418,7 +423,16 @@ private void populateWritersFromLibraryParams() {
                 samHeaderParams.put(tagName, row.getField(tagName));
             }
 
-            final SAMFileWriterWrapper writer = buildSamFileWriter(new File(row.getField("OUTPUT")),
+            File outputFile = new File(row.getField("OUTPUT"));
+
+            // If we are processing a single tile we want to append the tile number to the output file name. This is
+            // done to avoid file overwrites if you are running tile based processing in parallel.
+            if(PROCESS_SINGLE_TILE != null){
+                outputFile = new File(outputFile.getParentFile(),
+                        PROCESS_SINGLE_TILE.toString() +"."+ outputFile.getName());
```

Comment:
`toString()` is unnecessary here, it will be called automatically because of the `+` operator